docs/concepts/composition.md

```diff
@@ -161,813 +101,157 @@ spec:
-    Kind:         MySQLServer
-    Name:         example-zrpgr
-    UID:          3afb903e-32db-4834-a6e7-31249212dca0
-    API Version:  database.azure.crossplane.io/v1alpha3
-    Kind:         MySQLServerFirewallRule
-    Name:         example-h4zjn
-    UID:          602c8412-7c33-4338-a3af-78166c17b1a0
-  Write Connection Secret To Ref:
-    Name:       example-mysqlinstance
-    Namespace:  infra-secrets
-Status:
-  Address:    example.mysql.database.azure.com
-  Admin DSN:  mysql://admin@example.mysql.database.azure.com:3306/my-database-name
-  Conditions:
-    Last Transition Time:  2020-05-15T06:56:46Z
-    Reason:                Resource is available for use
-    Status:                True
-    Type:                  Ready
-    Last Transition Time:  2020-05-15T06:53:16Z
-    Reason:                Successfully reconciled resource
-    Status:                True
-    Type:                  Synced
-  Connection Details:
-    Last Published Time:  2020-05-15T06:53:16Z
-Events:
-  Type    Reason                   Age                  From                                           Message
-  ----    ------                   ----                 ----                                           -------
-  Normal  SelectComposition        10s (x7 over 3m40s)  composite/compositemysqlinstances.example.org  Successfully selected composition
-  Normal  PublishConnectionSecret  10s (x7 over 3m40s)  composite/compositemysqlinstances.example.org  Successfully published connection details
-  Normal  ComposeResources         10s (x7 over 3m40s)  composite/compositemysqlinstances.example.org  Successfully composed resources
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.parameters.storageGB
+      toFieldPath: spec.forProvider.settings.dataDiskSizeGb
 ```
 
-### Creating a Composite Resource Claim
+The above `Composition` tells Crossplane that when someone creates an
+`XPostgreSQLInstance` XR Crossplane should create a `CloudSQLInstance` in
+response. The `storageGB` field of the `XPostgreSQLInstance` should be used to
+configure the `dataDiskSizeGb` field of the `CloudSQLInstance`. This is only a
+small subset of the functionality a `Composition` enables - take a look at the
+[reference page][xr-ref] to learn more.
 
-Composite resource claims represent a need for a particular kind of composite
-resource, for example the above `MySQLInstance`. Claims are a proxy for the kind
-of resource they claim, allowing platform customers to provision and consume an
-XR. A claim may request a pre-existing, statically provisioned XR or it may
-dynamically provision one on-demand.
+> We almost always talk about XRs composing Managed Resources, but actually an
+> XR can also compose other XRs to allow nested layers of abstraction. XRs don't
+> support composing arbitrary Kubernetes resources (e.g. Deployments, operators,
+> etc) directly but you can do so using our [Kubernetes][provider-kubernetes]
```

Comment:
should the provider-kubernetes and provider-helm links resolve to the GitHub repositories?

Reply:
They do for me. These are "footnote" style markdown links, which I feel makes the raw markdown more readable. See the `provider-kubernetes` link in the list of links at the bottom.